src/driver/amdxdna/amdxdna_drv.c

```diff
@@ -179,7 +179,7 @@ static int amdxdna_drm_get_info_ioctl(struct drm_device *dev, void *data, struct
 {
 	struct amdxdna_drm_get_info *args = data;
 	struct amdxdna_dev *xdna = to_xdna_dev(dev);
-	int ret;
+	int ret = {};
```

Comment:
Why not ret = 0?

Reply:
It's just what I'm used too as it will more often do the correct thing when I need it to... either should be fine I'd imagine, whatever the style of you're project is, I apologize I didn't spend that much time reading it over to get the typical convention used ;)

Reply:
Looking at the code closer, you can't even initialize it to 0. It should be -EINVAL. In this case, '{}' will not work.